runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -350,6 +354,10 @@ public void onMenuOptionClicked(MenuOptionClicked event)
 	@Subscribe
 	public void onChatMessage(ChatMessage event)
 	{
+		if (event.getMessage().equals(VENGEANCE_USED_MESSAGE) && event.getName().equals(client.getLocalPlayer().getName()))
```

Comment:
What is the chattype here? You may also be able to check that. Also this may not work with players with spaces in their names?

Reply:
Good catch. Changed to message equality and `ChatMessageType.PUBLIC`. Despite it being public, the message only displays in the chatbox for the player.

Reply:
I would guess this is something other than `PUBLIC`.

Reply:
`[Thread-5] DEBUG net.runelite.client.RuneLite - Chat message type PUBLIC: Taste vengeance!`
*shrugs*